packages/components/pie-checkbox/src/index.ts

```diff
@@ -183,11 +180,14 @@ export class PieCheckbox extends FormControlMixin(RtlMixin(LitElement)) implemen
             <label for="inputId" data-test-id="checkbox-component">
                 <span
                     class="c-checkbox-tick"
+                    ?isRTL=${this.isRTL}
```

Comment:
`isRtl` wouldn't be a valid HTML attribute here?

Reply:
I think in html we can add custom attributes without issues, they won't do anything without binding them to some logic. 

In our RTL mixin the example shows this as a valid use case: https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-webc-core/src/mixins/rtl/rtlMixin.ts#L44

We might need to update it to `data-isRTL` attribute to be safe 🤔

Reply:
Yeh, I think we'll need to, as they would fail HTML validation checks (that's why we're changing the styles over to use classes instead of the custom attributes)

Reply:
I'll create a follow up PR for this

Reply:
Here it is: https://github.com/justeattakeaway/pie/pull/1625